vendor/github.com/emicklei/go-restful-swagger12/swagger_webservice.go

```diff
@@ -59,13 +59,21 @@ var LogInfo = func(format string, v ...interface{}) {
 // InstallSwaggerService add the WebService that provides the API documentation of all services
 // conform the Swagger documentation specifcation. (https://github.com/wordnik/swagger-core/wiki).
 func InstallSwaggerService(aSwaggerConfig Config) {
-	RegisterSwaggerService(aSwaggerConfig, restful.DefaultContainer)
+	RegisterSwaggerService(aSwaggerConfig, nil, restful.DefaultContainer)
 }
 
 // RegisterSwaggerService add the WebService that provides the API documentation of all services
 // conform the Swagger documentation specifcation. (https://github.com/wordnik/swagger-core/wiki).
-func RegisterSwaggerService(config Config, wsContainer *restful.Container) {
+func RegisterSwaggerService(config Config, containers []*restful.Container, wsContainer *restful.Container) {
 	sws := newSwaggerService(config)
+	for _, container := range containers {
+		sws.AddWebServices(container.RegisteredWebServices())
```

Comment:
oh right, you have to collect all the services. Forget my previous question why we need all the containers.